README.md

```diff
@@ -700,6 +702,44 @@ In this example, all saves to `MyTable` will raise an `Dynamoid::Errors::StaleOb
 
 Calls to `update` and `update!` also increment the `lock_version`, however they do not check the existing value. This guarantees that a update operation will raise an exception in a concurrent save operation, however a save operation will never cause an update to fail. Thus, `update` is useful & safe only for doing atomic operations (e.g. increment a value, add/remove from a set, etc), but should not be used in a read-modify-write pattern.
 
+
+### Backoff strategies
+
+
+You can use several methods what do efficiently in batch mode like `.find_all` and `.import`.
+
+By some reasons, for instanse table throughput limit is exceeded, some items could be not processed.
+By default operations will be re-run to process this items.
```

Comment:
This is not the default, unless you turn on exponential backoff, right?  It is turned off by default?

Change to:
> When exponential backoff is configured operations will be rerun to process items that failed.

Reply:
> This is not the default, unless you turn on exponential backoff, right?

@pboling No. I changed default behaviour. Even if backoff isn't set we will process all the items but without any delay. Currently we just lose data at batch writing and reading so I would consider it as a bug fix as well.

Reply:
I am confused by this changeset.  It looks like the default for exponential backoff is `nil`, and thus it is not used by default.  Is that correct?

Reply:
@pboling As you wrote below `by default` means `built-in`. Will replace it everywhere.